docs/manual/technical_design.md

```diff
@@ -331,6 +333,58 @@ The restore reconciler runs the following subreconcilers:
 
 The `StartRestore` subreconciler starts a restore. If there is no active restore, this will run a `start` command in `fdbrestore`.
 
+## Interaction Between the Operator and the Pods
+
+The operator communicates with processes running inside of the FoundationDB pods at multiple stages in the reconciliation flow. The exact flow will depend on whether you are using split images (which is the current default) or unified images.
+
+### Split Image
```

Comment:
Should we rather name that "Sidecar Image" I know that this is ambiguous with the sidecar pattern but it's a little bit more specific than split image.

Reply:
I thought that "split" would be a better parallel to "unified", which is the way we've been describing the new image.